lldb/test/API/lang/swift/nserror/TestNSError.py

```diff
@@ -46,19 +54,26 @@ def nserror_commands(self):
             "thread list", STOPPED_DUE_TO_BREAKPOINT,
             substrs=['stopped', 'stop reason = breakpoint'])
 
+        substrs = [
+            '0 = " "',
+            '0 = "x+y"',
+            '1 = 0x',
+            'domain: "lldbrocks" - code: 3133079277 {',
+            'domain: "lldbrocks" - code: 0 {'
+        ]
+
+        if check_userInfo:
+            substrs.extend([
+                '_userInfo = 2 key/value pairs {',
+                    '[0] = {',
+                    '[1] = {',
+                        'key = "x"',
+                        'key = "y"',
+                        'value = 0',
+                        'value = 3',
+                        'value = 4'])
+
         self.expect(
             "frame variable -d run --ptr-depth=2",
```

Comment:
so we can't determine the type of the NSError object _even though_ we use `-d run`? That's different/worse than the other ObjC runtime issue...

Reply:
Yes with a slight correction, it's `userInfo` field (within `NSError`) that lldb doesn't determine the type of. It works for objc, but not swift.